test/unit-tests/tablecell.test.js

```diff
@@ -0,0 +1,330 @@
+
+describe(test_name, () => {
+  let filePath;
+
+  let validateEmitterEvent = function(event, expectedText) {
+    return new Promise(resolve => {
+      descEvent.once(event, eventData => {
+        expect(eventData).to.be.equal(expectedText);
+        resolve();
+      });
+    });
+  };
+
+  before(async () => {
+    let innerHtml = `
+      <article id="text__hr">
+      <header><h1>Horizontal rules</h1></header>
+      <div>
+        <hr>
+      </div>
+      <footer><p><a href="#top">[Top]</a></p></footer>
+    </article>
+    <article id="text__tables">
+            <header><h1>Tabular data</h1></header>
+            <table>
+              <caption>Table Caption</caption>
+              <thead>
+                <tr>
+                  <th>Table Heading 1</th>
+                  <th>Table Heading 2</th>
+                  <th>Table Heading 3</th>
+                  <th>Table Heading 4</th>
+                  <th>Table Heading 5</th>
+                </tr>
+              </thead>
+              <tfoot>
+                <tr>
+                  <th>Table Footer 1</th>
+                  <th>Table Footer 2</th>
+                  <th>Table Footer 3</th>
+                  <th>Table Footer 4</th>
+                  <th>Table Footer 5</th>
+                </tr>
+              </tfoot>
+              <tbody>
+                <tr>
+                  <td>Table Cell 1</td>
+                  <td>Table Cell 2</td>
+                  <td>Table Cell 3</td>
+                  <td id='lucky'>Table Cell 4</td>
+                  <td>Table Cell 5</td>
+                </tr>
+                <tr>
+                  <td>Table Cell 1</td>
+                  <td>Table Cell 2</td>
+                  <td>Table Cell 3</td>
+                  <td>Table Cell 4</td>
+                  <td>Table Cell 5</td>
+                </tr>
+                <tr>
+                  <td>Table Cell 1</td>
+                  <td>Table Cell 2</td>
+                  <td>Table Cell 3</td>
+                  <td>Table Cell 4</td>
+                  <td>Table Cell 5</td>
+                </tr>
+                <tr>
+                  <td>Table Cell 1</td>
+                  <td>Table Cell 2</td>
+                  <td>Table Cell 3</td>
+                  <td>Table Cell 4</td>
+                  <td>Table Cell 5</td>
+                </tr>
+              </tbody>
+            </table>
+            <footer><p><a href="#top">[Top]</a></p></footer>
+          </article>
+      `;
+    filePath = createHtml(innerHtml, test_name);
+    await openBrowser(openBrowserArgs);
+    await goto(filePath);
+    setConfig({
+      waitForNavigation: false,
+    });
+  });
+
+  after(async () => {
+    setConfig({
+      waitForNavigation: true,
+    });
+    await closeBrowser();
+    removeFile(filePath);
+  });
+
+  describe('using label as Table Caption', () => {
+    it('test tableCell exists()', async () => {
+      expect(
+        await tableCell(
+          {
+            row: '1',
```

Comment:
Prefer the api to be dealing with numbers instead of string can this be 
```
tableCell({row: 1, col: 1})
```

Reply:
I was eagerly waiting for some review comments. Sure. Long time @zabil  :)